### DISCUSSION TITLE
Difference between simplelogin/app-ci and simplelogin/app

### DISCUSSION THREAD

**Discussion by wforumw** [simple-login/app#1614] [Q&A]
I have a small question.
What is the diffence between  **simplelogin/app-ci** and **simplelogin/app** on hub.docker.com
Thanks in advance!

**Comment by nguyenkims** [comment]
The app-ci one is used for running app.simplelogin.io whereas the app one is used for self hosting. The app-ci is usually ahead of app.

**Reply by wforumw** [reply]
Thanks for the quick response!
Can you also use the app-ci for self hosting?

**Reply by t2noob** [reply]
Yes, but I have read that there might be some issues. I'm running the latest app-ci and haven't ran into any issues yet. But you could always go back to previous build. Take it as a grain of salt, depending on your setup, you might have issues.

**Reply by wforumw** [reply]
Thanks again, is there a way to see the difference between both?
